Composer/packages/server/assets/projects/ActionsSample/ComposerDialogs/Actions/Actions.dialog

```diff
@@ -16,7 +16,7 @@
         {
           "$type": "Microsoft.SendActivity",
           "$designer": {
-            "id": "522200"
+            "id": "166131"
           },
           "activity": "${bfdactivity-522200()}"
         },
```

Comment:
Just curious about why many `$designer.id` fields are changed? Are those changes intended?

Reply:
Another question is why the designer id here is not equal to lg template's id?

Reply:
> Another question is why the designer id here is not equal to lg template's id?



> Just curious about why many `$designer.id` fields are changed? Are those changes intended?

No, it's not intended. It just changed after I build it. It should be reverted to the original value. Will fix it soon.